packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -131,22 +131,21 @@ function BlockPopoverInbetween( {
 						? previousRect.height
 						: nextRect.height;
 
+					if ( previousRect && nextRect ) {
+						width = nextRect.left - previousRect.right;
+						// Avoid a negative width when the next rect is on
+						// the next line.
+						width = Math.max( width, 0 );
+					}
+
 					if ( isRTL() ) {
 						// non vertical, rtl
 						left = nextRect ? nextRect.right : previousRect.left;
-						width =
```

Comment:
I don't understand why you removed the isRTL case?

Reply:
I misread it, I though it's the same and only `left` was different.

Reply:
Fixed